app/component/itinerary/NaviDestination.js

```diff
@@ -1,21 +1,38 @@
 import React from 'react';
-import { intlShape } from 'react-intl';
-import { legShape } from '../../util/shapes';
+import { FormattedMessage, intlShape } from 'react-intl';
+import { legShape, configShape } from '../../util/shapes';
+import { displayDistance } from '../../util/geo-utils';
+import { durationToString } from '../../util/timeUtils';
 
-function NaviDestination({ leg }) {
+function NaviDestination({ leg }, { config, intl }) {
   const { stop, rentalVehicle, vehicleParking, vehicleRentalStation, name } =
     leg.to;
-  const plat = `Laituri: ${stop?.platformCode}`;
+  const { distance, duration } = leg;
+
   return (
     <div className="navileg-destination-details">
       <div>
         {stop?.name || name} &nbsp;
         {stop?.code} &nbsp;
-        {stop?.platformCode && plat}
+        {stop?.platformCode && (
```

Comment:
Would existing PlatformNumber component do the job?

Reply:
No:
![image](https://github.com/user-attachments/assets/34beb117-dd5f-4837-90c4-6d5135dc7899)